mytestvueapp.client/src/router/index.ts

```diff
@@ -18,6 +18,11 @@ const router = createRouter({
       name: "Painter View",
       component: () => import("../views/PainterView.vue"),
     },
+    {
+      path: "/prompt",
```

Comment:
can we make this more descriptive? maybe call the path something like /create or /new?

Reply:
changed to /new